WORKSPACE

```diff
@@ -39,10 +39,10 @@ apple_support_dependencies()
 
 http_archive(
     name = "capnp-cpp",
-    sha256 = "d2cac87fabbb9dfd38a20903158b533eb38cd369f8d5fe89b65fef3f75099cad",
-    strip_prefix = "capnproto-capnproto-9084594/c++",
+    integrity = "sha256-kVnOrg/h0eq37kBsFEHTUZax20JTR4/1eDy+kRY7wD8=",
```

Comment:
@smerritt just curious about this change, all the other deps still use `sha256 = `. Is this better, and if so should we switch the other deps as well?

Reply:
It looks equivalent, like it's a sha256 but base64 encoded (and with a type specifier).

I think this is different because we upgraded to a new Bazel version. The old version, if no checksum was specified, would say "hey, put in `sha256 = "abc123..."` for assurance that it hasn't changed", and then I would put that line in. The new Bazel version says "hey, put in `integrity = "sha256-asdfZr+blah"`", so I did that.